collect_app/src/test/java/org/odk/collect/android/preferences/AdminPreferencesActivityTest.java

```diff
@@ -42,7 +44,7 @@ public void shouldUpdateAdminSharedPreferences() throws NullPointerException {
         for (String adminKey : AdminKeys.ALL_KEYS) {
             Preference preference = adminPreferencesFragment.findPreference(adminKey);
             if (preference instanceof CheckBoxPreference) {
-                System.out.println("Testing " + adminKey);
+                Timber.d("Testing " + adminKey);
```

Comment:
some other code checker will complain that it doesn't use String template. Timber.d("Testing %s", adminKey);